reduct/boolean-reduct/redundant.metta

```diff
@@ -0,0 +1,27 @@
+
+; This function assumes it receives a dominant set and the current expression as parameters. 
+; It then removes redundant guard sets from the current expression that already exist in the dominant set. 
+; The dominant set should be the set of guard sets of the parent AND nodes for the current AND node.
+; This function also can be used for lCCSUBTRACT transformation with command set and current expression given as a parameter.
+; The function uses isMember from general helpers functions.
+
+
+(: setDifference (-> Expression Expression Expression))
```

Comment:
Dude. This function shouldn't be named `setDifference` it can confuse people in the long run. Can you rename it?

Reply:
@Yagth this is what it says on the RTE function and what it does. its a function given for redundant transformation. should i call it `removeRedundant`.

Reply:
Best if you can do that, since it's actions corresponds to that transformation :)